File: WebBiblioteca_Api/BibliotecaApiDLL/secao/Secao.cs
namespace BibliotecaApiDLL.secao
{
    public class Secao
    {
        public Secao(int codSeCao, string descricaoSecao, string codLocal)
        {
            CodSecao = codSeCao;
            DescricaoSecao = descricaoSecao;
            DescricaoLocal = codLocal;
        }

        public int CodSecao { get; set; }
        public string DescricaoSecao { get; set; }
        public string DescricaoLocal { get; set; }
    }
}

File: WebBibliotecaApp/src/Objects/Secao.ts
export class Secao {
    codSecao: number = 0;
    descricaoSecao: string = '';
    descricaoLocal: string = '';

    constructor(codSecao: number, descricaoLocal: string, descricaoSecao: string){
        this.codSecao = codSecao;
        this.descricaoLocal = descricaoLocal;
        this.descricaoSecao = descricaoSecao;
    }
}
